data/json/items/ammo/762x25.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "id": "762_25",
+    "type": "AMMO",
+    "name": "7.62x25mm JHP",
+    "name_plural": "7.62x25mm JHP",
+    "description": "A commercial version of the 7.62x25mm cartridge created for the armed forces of Soviet Russia.  It was derived from the 7.63x25mm cartridge used by the C96 pistol.",
+    "weight": 6,
```

Comment:
`weight: 8` as a cartridge is bigger than a comparable `weight: 7` 9mm cartridge